boards/mega-xplained/Makefile.include

```diff
@@ -1,6 +1,11 @@
 # define the cpu used by the Mega Xplained board
 export CPU = atmega1284p
 
+# TODO is there a bootloader on the flash ? I do not understand...
+# https://www.microchip.com/DevelopmentTools/ProductDetails/atmega1284p-xpld
+# ARDUINO_BOOTLOADER_SIZE = 2K
+ROM_RESERVED = $(ARDUINO_BOOTLOADER_SIZE)
```

Comment:
I could not find the bootloader size in the official docs. I did check the fuse settings for my mega-xplained and it is set to a boot size of 2048 words, which corresponds to 4KB (I assume that 4K is 4096 here and not 4000).

Currently the bootloader on mega-xplained is being clobbered rather than being used, due to a few things not being properly supported. Does this affect reserving ROM?

Also, this and a few of the other boards are not Arduinos, so you might want to change the name of ```ARDUINO_BOOTLOADER_SIZE``` to avoid confusion.

Reply:
Yes, on the mega-xplained the bootloader is not being used since it requires a very old programmer which is not available for Linux and afaik is not free. It is only included in some (old) versions of Atmel Studio.

I'd vote in favor to not taking it into account, how ever we should put a big warning somewhere indicating that the bootloader is being removed.

Reply:
I would rather say take the bootloader into account even if we are not using it.
So if a user is using his board both on windows and linux it works, if he uses only on Linux, he wastes the last 4k that are not used as he is not using the BL.

Reply:
That sounds okay, or even just ```BOOTLOADER_SIZE```.